src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const trimmedStyles = sourceString.split(';').map((style) => style.trim());
```

Comment:
[CODE KNOWLEDGE] - It's unnecessary to make a copy of the array before performing operations like `map` or `filter`, as these methods do not mutate the original array. You can chain these methods directly after the `split`.